docs/rules/require-meta-docs-url.md

```diff
@@ -96,29 +96,54 @@ module.exports = {
 }
 ```
 
+```js
+// eslint.config.js
+import eslintPlugin from 'eslint-plugin-eslint-plugin';
+
+export default [
+  {
+    plugins: { 'eslint-plugin': eslintPlugin },
+    rules: {
+      'eslint-plugin/require-meta-docs-url': [
+        'error',
+        {
+          pattern:
+            'https://github.com/eslint-community/eslint-plugin-eslint-plugin/blob/master/docs/rules/{{name}}.md',
+        },
+      ],
+    },
+  },
+];
+```
+
 If you set the `pattern` option, this rule adds `meta.docs.url` property automatically when you execute `eslint --fix` command.
 
 ## Version specific URL
 
-If you want to enforce version-specific URLs, it's feasible easily with `.eslintrc.js` and `npm version <type>` script.
+If you want to enforce version-specific URLs, it's feasible easily with `eslint.config.js` and `npm version <type>` script.
 For example:
 
-**.eslintrc.js**:
+**eslint.config.js**:
 
 ```js
-// const version = require("./package.json").version;
-
-module.exports = {
-  plugins: ['eslint-plugin'],
-  rules: {
-    'eslint-plugin/require-meta-docs-url': [
-      'error',
-      {
-        pattern: `path/to/v${version}/docs/rules/{{name}}.md`,
-      },
-    ],
+import eslintPlugin from 'eslint-plugin-eslint-plugin';
+import packageMetadata from './package.json' with { type: 'json' };
```

Comment:
> Error:   130:29  error  Can't resolve './package.json' in '/home/runner/work/eslint-plugin-eslint-plugin/eslint-plugin-eslint-plugin/docs/rules/require-meta-docs-url.md'  n/no-missing-import

You can disable that lint rule for this file if necessary.

Reply:
Example of disabling a lint rule for markdown code sample:

https://github.com/ember-cli/eslint-plugin-ember/blob/ea0e3802b9971884eab5092e07730e4a0b04d9de/eslint.config.js#L172

Reply:
Ah, I had `'n/no-missing-imports'` instead of `'n/no-missing-import'`  doh

Reply:
Looks good now